test/repl.jl

```diff
@@ -116,19 +116,19 @@ if !is_windows() || Sys.windows_version() >= Sys.WINDOWS_VISTA_VER
     # Issue #10222
     # Test ignoring insert key in standard and prefix search modes
     write(stdin_write, "\e[2h\e[2h\n") # insert (VT100-style)
-    @test search(readline(stdout_read), "[2h") == 0:-1
-    readline(stdout_read)
+    @test search(readline(stdout_read, false), "[2h") == 0:-1
+    readline(stdout_read, false)
     write(stdin_write, "\e[2~\e[2~\n") # insert (VT220-style)
-    @test search(readline(stdout_read), "[2~") == 0:-1
-    readline(stdout_read)
+    @test search(readline(stdout_read, false), "[2~") == 0:-1
+    readline(stdout_read, false)
     write(stdin_write, "1+1\n") # populate history with a trivial input
-    readline(stdout_read)
+    readline(stdout_read, false)
     write(stdin_write, "\e[A\e[2h\n") # up arrow, insert (VT100-style)
-    readline(stdout_read)
-    readline(stdout_read)
+    readline(stdout_read, false)
+    readline(stdout_read, false)
     write(stdin_write, "\e[A\e[2~\n") # up arrow, insert (VT220-style)
-    readline(stdout_read)
-    readline(stdout_read)
+    readline(stdout_read, false)
+    readline(stdout_read, false)
```

Comment:
would it also be a good idea to test the default behavior with `readline(....)`    without line endings or is that already being tested?